lib/close_old_pull_requests.rb

```diff
@@ -24,11 +22,11 @@ def initialize(pull_requests)
     end
 
     def outdated
```

Comment:
I think it would be helpful to have some documentation of the return value of this method now that it's more complex